instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -21,9 +45,9 @@ export class PostContainer extends Component {
                 </div>
 
                 <div className="likes">
-                    <FontAwesome className="heart" name='heart' />
-                    <p>{this.props.data.likes}</p>
-                    <FontAwesome className="comment" name='comment' />
+                    <FontAwesome className="heart" name='heart' onClick={this.selected} />
+                    <p>{this.state.likes}</p>
+                    <FontAwesome className="comment" name='comment' onClick={this.selected} />
```

Comment:
because you placed your selected method with an onClick on BOTH heart and comment.. both of those icons increment/decrement the likes :)